docs/source/index.rst

```diff
@@ -24,3 +24,22 @@ Crossroad
 
 This documentation corresponds to implementation in version |version|,
 documentation was generated on |today|.
+
+
+Creating New Solvers
+####################
+
+A. Firstly identify the upstream image that you wish to develop a new solver from. Create an overlay for that new solver in [`thoth-station/s2i-thoth`](https://github.com/thoth-station/s2i-thoth). Each solver overlay should have 4 files.
+ 1. A `Dockerfile` to build the image from. This should be based off of the other examples available in the (s2i-thoth-repo)[https://github.com/thoth-station/s2i-thoth].
+ 2. A `requirements.in` stating the required dependencies.
+ 3. A `Requirements.txt` which can be generated from pip compile using the following command once the `requirements.in` file is present: `pip-compile --generate-hashes requirements.in`.
+ 4. An `s2i_assemble.patch` file. This will be used to patche the upstream image's s2i_assemble file with the dependencies required by thoth-station.
```

Comment:
```suggestion
 4. An `s2i_assemble.patch` file. This will be used to patch the upstream image's s2i_assemble file with the dependencies required by thoth-station.
```